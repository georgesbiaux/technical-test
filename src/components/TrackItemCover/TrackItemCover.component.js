import styled from 'styled-components';

const TrackItemCover = styled.div`
  width: 50px;
  height: 50px;
  border-radius: 4px;
  background-image: url('${props => props.pictureUrl}');
  background-size: cover;
`;

export default TrackItemCover;
