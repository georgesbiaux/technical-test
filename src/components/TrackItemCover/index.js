export { default } from './TrackItemCover.component';
