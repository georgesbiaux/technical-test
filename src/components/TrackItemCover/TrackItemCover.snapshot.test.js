import React from 'react';
import { render } from 'enzyme';
import toJSON from 'enzyme-to-json';
import TrackItemCover from './TrackItemCover.component';

describe('<TrackItemCover>', () => {
  it('should render correctly', () => {
    const props = {
      pictureUrl:
        'https://i.scdn.co/image/adcf79a6adae4e1a44d0e34860cc103e28ee4372',
    };
    const tree = render(<TrackItemCover {...props} />);
    expect(toJSON(tree)).toMatchSnapshot();
  });
});
