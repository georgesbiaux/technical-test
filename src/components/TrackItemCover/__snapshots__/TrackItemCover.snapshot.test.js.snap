// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackItemCover> should render correctly 1`] = `
<div
  class="sc-bdVaJa jYXbEs"
>
  
</div>
`;
